core/ledger/listing_balances.go

```diff
@@ -0,0 +1,270 @@
+package ledger
+
+import (
+	"encoding/hex"
+	"sort"
+
+	"github.com/anoideaopen/foundation/core/balance"
+	"github.com/anoideaopen/foundation/core/types/big"
+	"github.com/hyperledger/fabric-chaincode-go/shim"
+	"github.com/hyperledger/fabric-protos-go/ledger/queryresult"
+)
+
+// Record - one entry
+type Record struct {
```

Comment:
pls add more specific record name to prevent collisions like ledger.Record. maybe ledger.BalanceRecord or smth like that